packages/components/src/accordion/use-accordion.ts

```diff
@@ -185,9 +166,21 @@ export interface UseAccordionItemProps {
    */
   isFocusable?: boolean
   /**
-   * A unique id for the accordion item.
+   * The unique id of the accordion item.
    */
   id?: string
+  /**
+   * Unique value the accordion item.
+   */
+  value?: string
+}
+
+function getAllItems(root: HTMLElement | null) {
+  return queryAll(root, "[aria-controls]:not([disabled])")
+}
+
+function makeId(type: string, id: string, value: string) {
+  return `accordion-${type}-${id}-${value}`
```

Comment:
@segunadebayo the id might break when value includes spaces or unsupported characters, is there a normalisation function available?

Reply:
How about we let the users handle this? By default react ids are escaped properly. We can also use a string escape library

Reply:
Escaping whitespace chars now, all other chars should be pretty much valid.